stl/inc/flat_set

```diff
@@ -625,14 +632,13 @@ private:
         if constexpr (!_Presorted) {
             _STD sort(_Old_end, _New_end, _Comp);
         } else {
-            _STL_ASSERT(_STD is_sorted(_Old_end, _New_end, _Comp), "Input was not sorted!");
+            _STL_ASSERT(_Check_sorted(_Old_end, _New_end), _Msg_not_sorted);

```

Comment:
This check is enhanced. It didn't check for unique-ness for `flat_set` previously.